<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPSrcGUI</class>
 <widget class="RollupWidget" name="TCPSrcGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>443</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TCP Sample Source</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>201</width>
     <height>142</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Sample Format</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="sampleFormat">
      <property name="toolTip">
       <string>Sample format</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>S16LE I/Q</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>S16LE NFM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>S16LE SSB</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="rfBandwidth">
      <property name="toolTip">
       <string>Channel bandwidth</string>
      </property>
      <property name="text">
       <string>32000</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>RF Bandwidth (Hz)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Samplerate (Hz)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>TCP Port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="sampleRate">
      <property name="toolTip">
       <string>Data stream sample rate</string>
      </property>
      <property name="text">
       <string>48000</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="tcpPort">
      <property name="toolTip">
       <string>TCP port number</string>
      </property>
      <property name="text">
       <string>9999</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="applyBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Apply changes</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="DeltaFrequencyLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="deltaFrequencyLabel">
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Df</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ValueDialZ" name="deltaFrequency" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Demod shift frequency from center in Hz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deltaUnits">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>26</red>
              <green>26</green>
              <blue>26</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>26</red>
              <green>26</green>
              <blue>26</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>118</red>
              <green>118</green>
              <blue>117</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Hz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="ChannelPowerLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="channelPower">
        <property name="toolTip">
         <string>Channel power</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="channelPowerUnits">
        <property name="text">
         <string> dB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="BoostLayout">
      <item>
       <widget class="QLabel" name="volumeLabel">
        <property name="text">
         <string>Vol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volume">
        <property name="toolTip">
         <string>Amplitude boost</string>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="volumeText">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="spectrumBox" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>160</y>
     <width>231</width>
     <height>156</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Channel Spectrum</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="GLSpectrum" name="glSpectrum" native="true">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GLSpectrumGUI" name="spectrumGUI" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="connectedClientsBox" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>330</y>
     <width>274</width>
     <height>101</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Connected Clients (0)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTreeWidget" name="connections">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>IP:Port</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLSpectrum</class>
   <extends>QWidget</extends>
   <header>gui/glspectrum.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLSpectrumGUI</class>
   <extends>QWidget</extends>
   <header>gui/glspectrumgui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ValueDialZ</class>
   <extends>QWidget</extends>
   <header>gui/valuedialz.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sampleFormat</tabstop>
  <tabstop>tcpPort</tabstop>
  <tabstop>sampleRate</tabstop>
  <tabstop>rfBandwidth</tabstop>
  <tabstop>applyBtn</tabstop>
  <tabstop>connections</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
